examples/specs/box-plot_minmax_1D_horizontal.vl.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
As said on slack, let's not remove examples so we have test to prevent regression and ensure they still produce vega and SVG outputs fine.

Reply:
One sec